specification/security/resource-manager/readme.md

```diff
@@ -154,12 +154,12 @@ input-file:
 - Microsoft.Security/preview/2019-01-01-preview/regulatoryCompliance.json
 - Microsoft.Security/preview/2019-01-01-preview/serverVulnerabilityAssessments.json
 - Microsoft.Security/preview/2019-01-01-preview/subAssessments.json
-- Microsoft.Security/preview/2019-01-01-preview/automations.json

```

Comment:
I'm not sure why this is deleted and moved to the override-info. This may cause the errors you get in the sdks?